hetmech/test_diffusion.py

```diff
@@ -52,12 +52,16 @@ def test_dual_normalize_column_damping(self, exponent, dtype, copy):
         assert numpy.allclose(numpy.transpose(expect), matrix)
 
         # Create the matrix expected for simultaneous dual normalization
-        expect = [ [(1/3**exponent) / (1/3**exponent + 1/2**exponent + 1)**exponent,
-                    (1/3**exponent) / (1/3**exponent + 1/2**exponent)**exponent,
-                    (1/3**exponent) / (1/3**exponent)**exponent],
-                [(1/2**exponent) / (1/3**exponent + 1/2**exponent + 1)**exponent,
-                 (1/2**exponent) / (1/3**exponent + 1/2**exponent)**exponent, 0],
-                [1 / (1/3**exponent + 1/2**exponent + 1)**exponent, 0, 0],
+        expect = [
```

Comment:
Perhaps this would be more readable if you prefaced it with `e = exponent`, so it would be shorter?